README.md

```diff
@@ -17,12 +17,13 @@ On this page you will find everything you need to get started using your dev kit
 - [Build a custom solution using Advanced code (VS Code + AML)](https://github.com/microsoft/Project-Santa-Cruz-Private-Preview/blob/main/Sample-Scripts-and-Notebooks/Official/MobileNetV2SSDL_TrainingonVSCodeIDE/readme.md)
 
 ## 4. Secure Your Model and Data
-
-* [Setup a secured AI model locker](user-guides/secured_locker/deployment.md)
```

Comment:
delete secured

Reply:
Thanks. Removed secured in the new revision.